src/index.html

```diff
@@ -5,8 +5,260 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <a href="#home"><img class="header__logo" src="images/logo.png" alt="logo"></a>
+      <nav class="header__nav" id="nav">
+        <a href="#apple" class="header__link is-active"><ul class="text">apple</ul></a>
```

Comment:
зачем ты дополнительно оборачиваешь в ul? Из-за этого и отступы лишние в тестах
```suggestion
        <a href="#apple" class="header__link is-active">  
          apple
        </a>
```

Reply:
Thanks